### DISCUSSION TITLE
NO QOS DATA WIFI 6E

### DISCUSSION THREAD

**Discussion by HAlover99** [aircrack-ng/aircrack-ng#2605] [General]
Dear Community,

I'm tried sniffing the packet on lasted standard ( WIFI6E ) but show only beacon and rts-cts packets, all test are made with channel 1 33 and 65 at 160Mhzm, btw same Hardware works fine with 2.4Ghz and 5Ghz.
the chipset of card is mediatek (mt7921au) and the firmware already patched as suggested by Monitor-USB guide.
Someone could you help me to understand the issue?

**Comment by Mister-X-** [comment]
Airodump-ng defaults to 20MHz, so traffic other than what is sent at the low speeds isn't visible.

While it supports 20 and 40MHz, there will be changes to bring larger channels soon.

**Reply by Mister-X-** [reply]
As a temporary workaround, you can try once airodump-ng is locked on that channel to change the settings with `iw` to set the channel to 160MHz.

**Comment by HAlover99** [comment]
Hi mister x

Already use iw to set 160mhz but the result it's the same. 
only to clarify beacon and cts-rts are visible, missing only data qos 

EDIT: sniffing working fine only if not use channel bonding (20mhz)
Fronte my point of view root cause are driver or firmware

**Reply by Mister-X-** [reply]
There can be a few possibilities:
- You could be limited by the amount of streams the adapter can handle. Traffic with higher amount of streams won't be visible. 
- it may very well be due to beamforming, and not directed towards you.
- The traffic is a more recent modulation than what your adapter supports (802.11ax when the adapter is 802.11ac for example).
- Limitation of the driver/firmware

**Reply by HAlover99** [reply]
The adapter Support 802.11ax 160mhz 2x2 (netgear a8000)
Enviroments is controlled I have more STA and more AP.
I think that the issue is the driver/firmware...
